CDP4Common/Exceptions/ModelWarningException.cs

```diff
@@ -0,0 +1,82 @@
+﻿// -------------------------------------------------------------------------------------------------------------------------------
+// <copyright file="ModelWarningException.cs" company="RHEA System S.A.">
+//    Copyright (c) 2015-2023 RHEA System S.A.
+//
+//    Author: Sam Gerené, Merlin Bieze, Alex Vorobiev, Naron Phou, Alexandervan Delft, Nathanael Smiechowski, Ahmed Abulwafa Ahmed
+//
+//    This file is part of COMET-SDK Community Edition
+//
+//    The COMET-SDK Community Edition is free software; you can redistribute it and/or
+//    modify it under the terms of the GNU Lesser General Public
+//    License as published by the Free Software Foundation; either
+//    version 3 of the License, or (at your option) any later version.
+//
+//    The COMET-SDK Community Edition is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+//    Lesser General Public License for more details.
+//
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with this program; if not, write to the Free Software Foundation,
+//    Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+// </copyright>
+// -------------------------------------------------------------------------------------------------------------------------------
+
+namespace CDP4Common.Exceptions
+{
+    using System;
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// The COMET model warning exception is an exception that indicates that a model migt contain and model data might be corrupted.
```

Comment:
spelling errors and the sentence does not flow, please improve and make more verbose